pkg/plugin/javascript/dashboard_delete.go

```diff
@@ -38,13 +38,24 @@ func (d *DashboardDelete) Name() string {
 // delete is unsuccessful, it will throw a javascript exception.
 func (d *DashboardDelete) Call(ctx context.Context, vm *goja.Runtime) func(c goja.FunctionCall) goja.Value {
 	return func(c goja.FunctionCall) goja.Value {
+		var newCtx context.Context = context.WithValue(ctx, CloneKey, nil)
```

Comment:
This is confusing at a glance, I'd rather see

    // or optionally ctx, _ :=
    ctx, cancel := context.WithCancel(ctx)
    defer cancel()